### PR TITLE
feat: avatar and avatar group components added

### PR DESCRIPTION
**Pull request by Yhuley**
https://www.figma.com/file/NAlqiKft9MuxX3QmGQCleQ/Valcori_Application?node-id=6994%3A110149

### CODE REVIEW

**Comment by lazzystep** [comment]
Could you please provide more info about it. Ideally link to trello ticket as well. Thanks in advance!

**Comment by vlacher12** [comment]
Have a look at https://mui.com/components/avatars/ . Probably, we can find some inspiration there. AvatarGroup also could be implemented

**Review by sergdeev — APPROVED** [review]

**Review by taras-venger — APPROVED** [review]

File: src/components/avatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          {`${firstName?.charAt(0)?.toUpperCase() || ''}${lastName?.charAt(0)?.toUpperCase() || ''}`}
+        </Text>
+      }
+      </StyledAvatar>
+    )
+  );
+
+  if (tooltip) {
+    return (
+      <Tooltip content={email || `${firstName || ''} ${lastName || ''}`} {...tooltipProps}>
```
[the remark this answers is not in the document]
Comment (sergdeev) [reply src/components/avatar/index.tsx:66]: comment from Sasha
кароч імейл і ім'я це залежить від контексту
коли відоме ім'я то тоді його прокидуватимемоʼ
коли невідоме тоді імейл прокидуватимемо

**Review by sergdeev — APPROVED** [review]

File: src/components/avatar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  const fontSize = getFontSize(theme, size);
+
+  const renderAvatar = () => (
+    avatar ? (
+      <StyledAvatarImg src={avatar} alt={avatarAlt} style={{ height: size, width: size }} className={className} />
+    ) : (
+      <StyledAvatar theme={theme} style={{ height: size, width: size }} className={className}>
+      {(firstName || lastName) &&
+        <Text size={fontSize} weight={theme.typography.weights.bold} color={theme.colors.base500}>
+          {`${firstName?.charAt(0)?.toUpperCase() || ''}${lastName?.charAt(0)?.toUpperCase() || ''}`}
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:57]: Could you pls have a look at MUI solution? We might use smth similar to it.
https://mui.com/material-ui/react-avatar/#fallbacks

File: src/components/avatar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      return theme.typography.sizes.xl;
+    default : 
+      return theme.typography.sizes.m;
+  }
+}
+
+export interface AvatarProps {
+  className?: string;
+  avatar?: string;
+  avatarAlt?: string;
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:27]: alt instead of avatarAlt

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/avatar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            alt={alt} 
+            style={{ height: size, width: size }} 
+            className={className} 
+            onError={() => setImageLoadedError(true)}
+            {...imgProps} 
+          />
+        ) : (
+          <StyledAvatar theme={theme} style={{ height: size, width: size }} className={className}>
+            <Text size={fontSize} weight={theme.typography.weights.bold} color={theme.colors.base500}>
+              {children || alt?.charAt(0) || <Icon icon={Icon.icons.profile} size={size / 2} style={{ display: 'flex' }} />}
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:64]: Could you pls replace `profile` icon by `user`? If there is no `user` icon -> feel free to add
image

Comment (vlacher12) [reply src/components/avatar/index.tsx:64]: @Yhuley

**Review by vlacher12 — CHANGES_REQUESTED** [review]

**Comment by vlacher12** [comment]
@Yhuley Fix conflcits pls

**Review by vlacher12 — APPROVED** [review]

**Comment by vlacher12** [comment]
@sergdeev @Yhuley ok to merge

**Pull request merged**